content/en/synthetics/guide/identify_synthetics_bots.md

```diff
@@ -39,23 +39,34 @@ https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
 
 You can also identify Datadog robots by using some of the **default headers** attached to the requests generated by Synthetic tests:
 
+### user-agent
+
+A `user-agent` header is added to all requests performed by Synthetic tests. 
+
 {{< tabs >}}
 {{% tab "Single and multistep API tests" %}}
 
-The following header is attached to all single and multistep API tests:
+For single and multistep API tests, the `user-agent` header is `Datadog/Synthetics`.
 
-`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{% /tab %}}
+{{% tab "Browser tests" %}}
 
-A `user-agent: Datadog/Synthetics` is also added.
+For browser tests, the value of the `user-agent` header varies depending on the browser and device executing the test. The `user-agent` header value however always ends with `Datadog/Synthetics` to allow you to easily identify Synthetic tests.
 
 {{% /tab %}}
-{{% tab "Browser tests" %}}
+{{< /tabs >}}
 
-The following header is attached to all browser tests:
+### sec-datadog
 
-`Sec-Datadog: Request sent by a Datadog Synthetics Browser Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
+{{< tabs >}}
+{{% tab "Single and multistep API tests" %}}
+
+`sec-datadog: Request sent by a Datadog Synthetics API Test (https://docs.datadoghq.com/synthetics/) - test_id: <SYNTHETIC_TEST_PUBLIC_ID>`
```

Comment:
This is taking up multiple lines on the docs page. Use 3 ticks for a code block.

Reply:
Fixed - resolving comment.